schema/omega-services.async.api.yaml

```diff
@@ -48,6 +48,17 @@ channels:
         oneOf:
           - $ref: '#/components/messages/getEditSet'
 
+  omega.editorial.web.application.instance.1:
+    description: The queue on which response messages will be sent.
+    publish:
```

Comment:
I think this should be `subscribe:`, as in this is the channel that client's will subscribe to receive their responses. They won't publish here.